go/worker/storage/p2p/diff/protocol.go

```diff
@@ -0,0 +1,54 @@
+// Package diff defines wire protocol together with client/server
+// implementations for the storage diff protocol, used for runtime block sync.
+package diff
+
+import (
+	"time"
+
+	"github.com/libp2p/go-libp2p/core"
+	"github.com/oasisprotocol/oasis-core/go/common/node"
+	"github.com/oasisprotocol/oasis-core/go/common/version"
+	"github.com/oasisprotocol/oasis-core/go/p2p/peermgmt"
+	"github.com/oasisprotocol/oasis-core/go/p2p/protocol"
+	"github.com/oasisprotocol/oasis-core/go/storage/api"
+)
+
+// StorageDiffProtocolID is a unique protocol identifier for the storage diff protocol.
+const StorageDiffProtocolID = "storagediff"
```

Comment:
One is `checkpoints` the other is `storagediff`? Maybe `checkpointsync` and `diffsync`? Or `storagecheckpoints` and `storagediffs`?